pwndbg/gdblib/typeinfo.py

```diff
@@ -17,23 +18,27 @@
 ptrsize: int
 
 
-def lookup_types(*types):
+def lookup_types(*types: str) -> gdb.Type:
+    exc = None
     for type_str in types:
         try:
             return gdb.lookup_type(type_str)
         except Exception as e:
             exc = e
-    raise exc
+    if exc is not None:
+        raise exc
+    else:
+        raise RuntimeError("No types provided to lookup.")
 
 
 def update():
     # Workaround for Rust stuff, see https://github.com/pwndbg/pwndbg/issues/855
     lang = gdb.execute("show language", to_string=True)
-    if "rust" not in lang:
+    if lang and "rust" not in lang:
```

Comment:
assume show lang always returns what we want

Reply:
fixed!